alpine/Dockerfile

```diff
@@ -5,11 +5,11 @@ LABEL org.label-schema.vcs-url="https://github.com/Dwolla/jenkins-agent-docker-c
 
 ENV JENKINS_HOME=/home/jenkins \
     JENKINS_AGENT=/usr/share/jenkins \
-    AGENT_VERSION=2.62.6
+    AGENT_VERSION=3.10
 
 COPY --from=nucleus / /
 
-RUN apk add --update --no-cache \
```

Comment:
Why did we remove `--update` here?

Reply:
Commented at the same time, but `update` has been renamed to `update-cache` in later versions.

Reply:
A `container-diff` run between an image with `--update-cache` and without it only results in some pip cache information changing, nothing about alpine packages.  It does push the responsibility of updating packages on `openjdk:8-jre-alpine`, so I'll add it back with the new flag name.